docs/features/instance-hierarchy-function-tr.md

```
@@ -0,0 +1,132 @@
+
+## Genel Bakış
+
+Hierarchy fonksiyonu, bir workflow instance'ının runtime hiyerarşisini özyinelemeli (recursive) ağaç yapısı olarak döndürür. Hem doğrudan hem de dolaylı alt subflow ve subprocess instance'larını içerir; iç içe geçmiş workflow ilişkilerinin tam görünürlüğünü sağlar.
+
+## API Uç Noktaları
+
+### Tek Instance Hiyerarşisi
+
+Belirli bir instance için tam hiyerarşi ağacını döndürür.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)